contrib/firefox/patches/flac-no-ffvpx.patch

```diff
@@ -4,22 +4,27 @@ diff --git dom/media/flac/FlacDecoder.cpp dom/media/flac/FlacDecoder.cpp
 index 53fc3c9937f7..b23771ab80fa 100644
 --- a/dom/media/flac/FlacDecoder.cpp
 +++ b/dom/media/flac/FlacDecoder.cpp
-@@ -7,6 +7,7 @@
+@@ -7,11 +7,22 @@
  #include "FlacDecoder.h"
  #include "MediaContainerType.h"
  #include "mozilla/StaticPrefs_media.h"
 +#include "PDMFactory.h"
  
  namespace mozilla {
  
-@@ -14,6 +15,10 @@ namespace mozilla {
- bool FlacDecoder::IsEnabled() {
- #ifdef MOZ_FFVPX
-   return StaticPrefs::media_flac_enabled();
+ /* static */
+-bool FlacDecoder::IsEnabled() { return StaticPrefs::media_flac_enabled(); }
++bool FlacDecoder::IsEnabled() {
++#ifdef MOZ_FFVPX
++  return StaticPrefs::media_flac_enabled();
```

Comment:
no idea if this rebase is correct or not since they removed the conditionals here, but probably fine